package/MDAnalysis/analysis/pca.py

```diff
@@ -121,11 +121,12 @@
 
 from MDAnalysis import Universe
 from MDAnalysis.analysis.align import _fit_to
-from MDAnalysis.lib.log import ProgressBar
+from MDAnalysis.lib.log import ProgressBar, ProgressBar_Dual
```

Comment:
ProgressBar_Dual is not used, please remove

Reply:
Done.